File: client/myStudents/src/types/Student.ts
export default class Student {
    id: number;
    name: string;
    matrikelnummer: string;
    semester: number;

    constructor(id: number, name: string, matrikelnummer: string, semester: number) {
        this.id = id;
        this.name = name;
        this.matrikelnummer = matrikelnummer;
        this.semester = semester
    }
}

File: api/models/Student.cs
namespace Abschlussprojekt.Models
{
    public class Student
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Matrikelnummer { get; set; } = string.Empty;
        public int Semester { get; set; }
        public string? University { get; set; }
    }
}